yarn.lock

```diff
@@ -879,13 +879,6 @@
   resolved "https://registry.yarnpkg.com/@bcoe/v8-coverage/-/v8-coverage-0.2.3.tgz#75a2e8b51cb758a7553d6804a5932d7aace75c39"
```

Comment:
Since this change was presumably automatically generated, and is too voluminous and opaque to review, I should ask instead: How was it generated?

Reply:
It's the lock of state from `yarn install` resolution of packages